contentcuration/contentcuration/frontend/channelEdit/vuex/clipboard/getters.js

```diff
@@ -461,92 +381,80 @@ export function getNextSelectionState(state, getters) {
   };
 }
 
-export function getCopyTrees(state, getters) {
+export function getCopyTrees(state, getters, rootState, rootGetters) {
   /**
    * Creates an array of "trees" of copy call arguments from the current selection
    *
-   * @param {string} target
+   * @param {string} id
+   * @param {Boolean} [ignoreSelection]
    * @return {[{ id: Number, deep: Boolean, target: string, children: [] }]}
    */
-  return function(rootId, ancestorId = null, ignoreSelection = false) {
-    function recurseForUnselectedIds(id, ancestorId) {
-      const selectionState = getters.getSelectionState(id, ancestorId);
+  return function(id, ignoreSelection = false) {
+    function recurseForUnselectedIds(id) {
+      const selectionState = getters.currentSelectionState(id);
       // Nothing is selected, so return early.
-      if (selectionState === SelectionFlags.NONE) {
-        const contentNode = getters.getClipboardNodeForRender(id, ancestorId);
+      if (!(selectionState & SelectionFlags.SELECTED)) {
```

Comment:
Be sure to check that it's not selected, not that it's selection state is None, because it could be indeterminate, but we would still want to excluded it in that case.